app/models/event_instance.rb

```diff
@@ -20,6 +20,18 @@ class EventInstance < ActiveRecord::Base
 
   validate :dont_update_after_finished, on: :update
 
+  def self.active_hangouts
+    select(&:live?)
+  end
+
+  def self.extract_yt_id(yt_video_url)
```

Comment:
I also wonder if we do this anywhere else - or if there is a library for it that would be even more generic?

I see three here: https://www.google.co.uk/search?q=extract+youtube+id+from+url

maybe https://github.com/luizpicolo/youtube_rails?